sanic/app.py

```diff
@@ -772,7 +831,7 @@ async def handle_request(self, request: Request):
                     )
 
                 # Run response handler
-                response = handler(request, **kwargs)
+                response = handler(request, **request.match_info)
```

Comment:
What do you mean? Basically I wanted to copy over the contents so the originals are not disturbed, but we allow for `request.match_info` to be modifiable (something I have been wanting for years).

Reply:
I mean, now the `handler` could use only the `request` argument ... I know it would break an undefined number of apps, so that would not be a good change. Even though it makes sense to be the way it is ... NM, I'm just thinking out loud I guess :sweat_smile: